Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_usb_ven.cpp

```diff
@@ -15,21 +15,19 @@ CWII_IPC_HLE_Device_usb_ven::CWII_IPC_HLE_Device_usb_ven(const u32 device_id,
 IPCCommandResult CWII_IPC_HLE_Device_usb_ven::IOCtlV(const IOSIOCtlVRequest& request)
 {
   request.Dump(GetDeviceName());
-  request.SetReturnValue(IPC_SUCCESS);
   return GetNoReply();
 }
 
 IPCCommandResult CWII_IPC_HLE_Device_usb_ven::IOCtl(const IOSIOCtlRequest& request)
 {
   request.Log(GetDeviceName(), LogTypes::OSHLE);
-  request.SetReturnValue(IPC_SUCCESS);
 
-  IPCCommandResult reply = GetDefaultReply();
+  IPCCommandResult reply = GetDefaultReply(IPC_SUCCESS);
```

Comment:
Yeah, I only made enough changes to get it to build, since the VEN stub is going to be replaced with a real implementation in #4408 ;) (which needs a rebase…)